src/test/java/io/jenkins/plugins/pipelinegraphview/utils/TestUtils.java

```diff
@@ -47,7 +47,7 @@ public static List<FlowNode> getNodesByDisplayName(WorkflowRun run, String displ
     graphTable.build();
     List<FlowNode> matchingNodes = new ArrayList<>();
     for (Row row : graphTable.getRows()) {
-      if (row.getDisplayName().equals(displayName)) {
+      if (row.getDisplayName().contains(" (" + displayName + ")")) {
```

Comment:
display name changed for some reason, not sure why, now says 'stage block (display name)'